### PR TITLE
Pratikshya TL

### PR DESCRIPTION
**Pull request by Pratikshya-Regmi**
This is the python file containing TL activity for GIS 714 class.

### CODE REVIEW

File: activities/pratikshya_TL.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        ["d.rast", "map=flow"]
+      ],
+      "base": "elev_lid792_1m",
+      "scanning_params": {
+        "smooth": 10,
+        "zexag": 2,
+        "numscans": 1,
+        "interpolate": true
+      },
+      "analyses": "pratikshya.py",
```
Comment (chaedri) [inline activities/pratikshya_TL.json]:
```suggestion
      "analyses": "pratikshya_TL.py",
```

**Comment by petrasovaa** [comment]
Please apply Caitlin's suggestion and use black to format the code (either install black locally or use https://black.vercel.app/).

File: activities/simple_example.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def run_slope(scanned_elev, env, **kwargs):
     gs.run_command("r.slope.aspect", elevation=scanned_elev, slope="slope", env=env)
 
 
+
```
Comment (petrasovaa) [inline activities/simple_example.py]: Please remove this file from the PR

File: activities/pratikshya_TL.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      ],
+      "base": "elev_lid792_1m",
+      "scanning_params": {
+        "smooth": 10,
+        "zexag": 2,
+        "numscans": 1,
+        "interpolate": true
+      },
+      "analyses": "pratikshya_TL.py",
+      "title": "SIMWE",
```
Comment (petrasovaa) [inline activities/pratikshya_TL.json:15]: Missing "author"

**Comment by petrasovaa** [comment]
You still have that extra file in there that needs to be removed.

**Pull request merged**